### ISSUE TITLE
AI responds multiple times But only the last reply will be displayed and recorded.

### ISSUE THREAD

**Issue by Czx1235** [waylaidwanderer/node-chatgpt-api#316]
**Describe the bug**
When AI responds multiple times at once,All replies except for the first reply's first character will disappear, and only the last reply will be displayed and recorded.
This is what I am looking for.But only for a moment
<img width="1102" alt="image" src="https://user-images.githubusercontent.com/116172450/232850769-662f0266-b9dd-4b14-8ed1-18e98da1e614.png">
And then it became like this, and all the previous content disappeared.The cache file is the same.
<img width="733" alt="image" src="https://user-images.githubusercontent.com/116172450/232851091-50d1e267-1007-4d65-9afa-78f8d2102960.png">

**To Reproduce**
1.Set some special system messages
2.Let AI reply multiple times at once

**Expected behavior**
All AI replies are displayed and stored

**Screenshots**
<img width="1266" alt="image" src="https://user-images.githubusercontent.com/116172450/232851886-26872fb0-e966-46df-b0c1-e2914867b6b7.png">

**Node.js version (please complete the following information):**
 - OS:Windows 11
 - Version 19

**Package version (please complete the following information):**
 - 1.35.0

**Additional context**
Allowing AI to respond multiple times at once with no more than 30 words can effectively bypass censorship.Maybe you can add this feature.

**Comment by Czx1235** [comment]
I tried to solve it myself.Thank you for your project.
Here is the result.
![image](https://user-images.githubusercontent.com/116172450/232992852-e5238aff-20b8-4fa8-9258-c8f9098679f5.png)
And here is my modification.
<img width="585" alt="image" src="https://user-images.githubusercontent.com/116172450/232993345-5e7fde7e-9ae9-452e-8c2c-09db953dbfc2.png">
<img width="602" alt="image" src="https://user-images.githubusercontent.com/116172450/232993474-fae50795-fdb4-4bb6-806a-a779655aa6ca.png">
Perhaps this will be useful for your project.
Anyway, thanks!

**Comment by waylaidwanderer** [comment]
Feel free to submit a pull request, but otherwise I'll keep this issue open until it's fixed.

**Comment by Czx1235** [comment]
I am not a professional, and there are still many bugs in the code I modified.So let's hope someone can fix it.

**Comment by kq514550** [comment]
A PR would still be valuable.

**Comment by Czx1235** [comment]
Yes, PR is still valuable, but the main bug is not this. I tried to fix some on my own, but I am a novice on GitHub, so I am not very good at using it like pull request.I can only paste the code I have modified, sorry.
Here.
![image](https://user-images.githubusercontent.com/116172450/235619710-582cc368-224d-4b09-830c-08a794d57367.png)
![image](https://user-images.githubusercontent.com/116172450/235619783-9f4f44cf-abcf-4ef5-bcef-271f508c92b7.png)
![image](https://user-images.githubusercontent.com/116172450/235619858-299b1ded-746d-4585-81cf-d080a363c828.png)
![image](https://user-images.githubusercontent.com/116172450/235619926-07c1dd96-5d4d-4d22-9716-53999bdc875a.png)
